src/locales/tr.json

```diff
@@ -10,14 +10,62 @@
   "footer": {
     "trademark": "Marka",
     "contact": "İletişim ",
-    "policies": "Kullanıcı Sözleşmesi",
+    "policies": "Politikalar",
     "games": "Oyunlar",
     "about": "Hakkında"
   },
-  "resourceSection": {
-    "pollution": "Çevre kirliliği",
-    "climate Change": "Mevsim değişimi",
-    "reduceReuseRecycle": "Azalt, tekrar kullan, geri dönüştür"
+  "login": {
+    "login": "Oturum aç",
+    "email": "Email",
+    "password": "Parola",
+    "forgotPassword": "Şifremi Unuttum?",
+    "new": "juniper yeni misiniz? Şimdi üye Ol",
+    "creat": "Üye Ol",
+    "oryoucan": "Ya da",
+    "withFacebook": "Facebook İle Bağlan",
+    "withGoogle": "Google İle Bağlan"
+  },
+  "passReset": {
+    "emailFormat": "Email biçimi Hatası",
+    "fillField": "Lütfen bu alanı doldurun",
+    "passwordReset": "Şifreyi yenile",
+    "email": "Email",
+    "backToLogin": "Geri Giriş Yap"
+  },
+  "register": {
+    "fullName": "lütfen tam adınızı giriniz",
+    "validEmail": "lütfen geçerli eposta adresini giriniz",
+    "enterAge": "Lütfen yaşını gir",
+    "passwordCharacter": "Şifre en az 6 karakter uzunluğunda olmalı",
+    "enterPassword": "Şifrenizi giriniz Lütfen",
```

Comment:
In Turkish sometimes word order can cause small meaning difference, you may use `"enterPassword": "Lütfen şifrenizi giriniz",`